docs/deployment/deploy_graphscope_on_existing_vineyard_cluster.md

```diff
@@ -0,0 +1,208 @@
+# Use an existing vineyard cluster with multiple GraphScope sessions
+
+If you have already deployed a vineyard cluster, you can easily deploy GraphScope on the existing cluster and reuse the vineyard data such as graph with several GraphScope sessions. This will allow you to load a graph to the existing vineyard cluster and then reuse it with multiple GraphScope sessions, without needing to deploy a separate vineyard cluster for each session.
+
+This doc provides step-by-step instructions on how to do this.
+
+## Prerequisites
+
+Before you begin, make sure you have the following:
+
+- Linux or macOS.
+- Python 3.7 ~ 3.11.
+
+## Install GraphScope Client
+
+The deployment of GraphScope is managed by python API, please make sure you have already installed the GraphScope python package.
+
+```bash
+python3 -m pip install graphscope-client
+```
+
+Refer to [Install GraphScope Client](./deploy_graphscope_on_self_managed_k8s.md#install-graphscope-client) for details.
+
+## Prepare a Kubernetes cluster
+
+If you don't have a Kubernetes cluster by hand, we recommend using [minikube](https://minikube.sigs.k8s.io/docs/start/).
+
+After installing it, you can start a Kubernetes cluster by
+
+```bash
+minikube start -p gs
+```
+
+Notice, if you don't use the default kubeconfig file, you need to set the environment variable `KUBECONFIG` to the path of the kubeconfig file. The next steps will use it to connect to the Kubernetes cluster.
+
+```bash
+export KUBECONFIG=your_kubeconfig_path
+```
+
+Check the nodes of the cluster by
+
+```bash
+$ kubectl get nodes -A
+NAME   STATUS   ROLES           AGE   VERSION
+gs     Ready    control-plane   18s   v1.24.1
+```
```

Comment:
Just refer to `docs/deployment/deploy_graphscope_on_self_managed_k8s.md` for the Kubernetes preparation and don't repeat the same instructions.

Reply:
Addressed.